### PR TITLE
[DOC] Fix missing te in the code example

### PR DESCRIPTION
**Pull request by insop**
This PR fixed the code example in this [page](https://tvm.apache.org/docs/deploy/hls.html).

- caused `AttributeError: module ‘tvm’ has no attribute ‘thread_axis’` error
- Fix based on [this thread](https://discuss.tvm.apache.org/t/attributeerror-module-tvm-has-no-attribute-thread-axis/6606)

Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

**Review by zhiics — APPROVED** [review]
The CI error is irrelevant. Please re-trigger when the master branch is fixed.

**Comment by insop** [comment]
> The CI error is irrelevant. Please re-trigger when the master branch is fixed.

Thank you @zhiics , how can I re-trigger The ci tests?

**Comment by comaniac** [comment]
You can push a dummy commit like this one: https://coderwall.com/p/vkdekq/git-commit-allow-empty

But please wait until #6572 is merged and the CI on master is green.

**Comment by insop** [comment]
> You can push a dummy commit like this one: https://coderwall.com/p/vkdekq/git-commit-allow-empty
> 
> But please wait until #6572 is merged and the CI on master is green.

Thank you @comaniac .
Got it, I understand about dummy commit. I asked because some of the Github/Jenkins setup I used, there is a way to trigger a new build without a dummy build.
Will wait for #6572 .

**Comment by insop** [comment]
> > You can push a dummy commit like this one: https://coderwall.com/p/vkdekq/git-commit-allow-empty
> > But please wait until #6572 is merged and the CI on master is green.
> 
> Thank you @comaniac .
> Got it, I understand about dummy commit. I asked because some of the Github/Jenkins setup I used, there is a way to trigger a new build without a dummy build.
> Will wait for #6572 .

I rebased to the master after #6572 , which triggered the build.

**Comment by comaniac** [comment]
Thanks @insop @zhiics

**Pull request merged**
